### PR TITLE
Adds a new network guide to create a custom container image.

### PR DESCRIPTION
**Pull request by iphutch**
Signed-off-by: ihutchin <ianeta.hutchinson@intel.com>

### CODE REVIEW

File: source/clear-linux/guides/network/custom-clear-container.rst [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+      # touch ./base/usr/share/clear/bundles/os-core
+      # touch ./base/usr/share/clear/bundles/os-core-update
+
+   .. note::
+
+      * `os-core` provides the minimal Linux namespace.
+      * `os-core-update` provides basic suite for running the |CL|
+        for iA Updater
+
+#. Optionally, additional bundles can be included with the base image.
```
Comment (rcaballeromx) [inline source/clear-linux/guides/network/custom-clear-container.rst]: The content in this step has the incorrect indentation and does not keep the 78 character line length.

File: source/clear-linux/guides/network/custom-clear-container.rst [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+
+   .. note::
+
+      * `os-core` provides the minimal Linux namespace.
+      * `os-core-update` provides basic suite for running the |CL|
+        for iA Updater
+
+#. Optionally, additional bundles can be included with the base image.   
+
+  Identify the desired bundles by going to the |CL| website's
```
Comment (rcaballeromx) [inline source/clear-linux/guides/network/custom-clear-container.rst]: Should be a level two ordered list along with the step on line 119.

File: source/clear-linux/guides/network/custom-clear-container.rst [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+
+#. Launch the built |CL| container.
+
+   .. code-block:: console
+
+      # docker run -it my-custom-clear-linux-container
+
+Manage bundles in a Clear Linux based container
+***********************************************
+Bundles can be added to and removed from an existing |CL| container by using
```
Comment (rcaballeromx) [inline source/clear-linux/guides/network/custom-clear-container.rst]: Unnecessary 'to' after 'added'.

File: source/clear-linux/guides/network/custom-clear-container.rst [hunk 4]
```diff
@@ -0,0 +1,378 @@
...
+      # docker run -it my-custom-clear-linux-container
+
+Manage bundles in a Clear Linux based container
+***********************************************
+Bundles can be added to and removed from an existing |CL| container by using 
+the `swupd` command in the Dockerfile.  
+
+Add a bundle (`swupd bundle-add`)
+---------------------------------
+Here is an example Dockerfile that shows adding the `pxe-server` bundle to the
```
Comment (rcaballeromx) [inline source/clear-linux/guides/network/custom-clear-container.rst]: The introductory texts of this section are all written in passive voice. Please change them all to something similar to:
This example Dockerfile shows how to add the `pxe-server` bundle to the previously created |CL| Docker image:

File: source/clear-linux/guides/network/custom-clear-container.rst [hunk 5]
```diff
@@ -0,0 +1,378 @@
...
+         Successfully tagged my-clearlinux-with-pxe-server-bundle:latest
+
+   .. note::
+
+      This `WARNING` message is expected and can be ignored because Systemd
+      doesn't run inside a container.
+
+Remove a bundle (`swupd bundle-remove`)
+---------------------------------------
+Here’s an example Dockerfile that shows removing the `pxe-server` bundle from the
```
Comment (rcaballeromx) [inline source/clear-linux/guides/network/custom-clear-container.rst]: See previous comment on introductory text..

**Comment by iphutch** [comment]
@rcaballeromx addressed all comments above.

File: source/clear-linux/guides/network/custom-clear-container.rst [hunk 6]
```diff
@@ -111,18 +111,19 @@ Build the base Clear Linux container image
       * `os-core-update` provides basic suite for running the |CL|
         for iA Updater
 
-#. Optionally, additional bundles can be included with the base image.   
+   #. Optionally, additional bundles can be included with the base image.
```
Comment (rcaballeromx) [inline source/clear-linux/guides/network/custom-clear-container.rst]: This line, did not have the indentation issue. Please render the doc to confirm that the numeration renders properly.

Comment (iphutch) [reply source/clear-linux/guides/network/custom-clear-container.rst]: Misunderstood your initial comment! Resolved!

**Pull request merged**